build.sc

```diff
@@ -87,71 +82,20 @@ object v {
     "-Xlint:infer-any",
     "-Xlint:missing-interpolator",
     "-language:reflectiveCalls",
-    s"-Wconf:${warnConf.mkString(",")}"
+    s"-Wconf:${scala2WarnConf.mkString(",")}"
   )
 }
 
```

Comment:
Are there some scala3CommonOptions we should be using?

Reply:
I don't think I've needed any yet, but if so I can add them in with the scala3 pr